src/xdocs/config_sizes.xml

```diff
@@ -325,6 +325,13 @@
             <th>default value</th>
             <th>since</th>
           </tr>
+          <tr>
+            <td>fileExtensions</td>
```

Comment:
New property due to file set.
This also means all files are processed instead of Java files by default.

Reply:
just mention this "migration notes" in issue.

Reply:
Done.